package.json

```diff
@@ -20,11 +20,14 @@
   },
   "homepage": "https://github.com/digitalbazaar/bedrock-key",
   "dependencies": {
-    "async": "^1.5.2"
+    "async": "^2.6.0"
+  },
+  "optionalDependencies": {
+    "bedrock-redis": "^3.0.0"
   },
   "peerDependencies": {
     "bedrock": "^1.4.1",
-    "bedrock-mongodb": "^3.0.0",
+    "bedrock-mongodb": "^4.0.0",
```

Comment:
May as well go to `5.0.0` here?

Reply:
I wonder if we should do a range for `bedrock-mongodb` ... like `"^3.0.0 - ^5.0.0"` ... is that's the right syntax.